api-reference/10 UI Components/dxSortable/1 Configuration/onDragStart.md

```diff
@@ -23,13 +23,13 @@ The UI component that raised the event.
 #include common-ref-eventparam
 
```

Comment:
Please update the following line above:
`A function that is called when drag gesture is initialized.`

`A function that is called when a drag gesture is initialized.`